Content.Shared/Eye/Blinding/Components/EyeClosingComponent.cs

```diff
@@ -12,6 +11,18 @@ namespace Content.Shared.Eye.Blinding.Components;
 [RegisterComponent, NetworkedComponent, AutoGenerateComponentState(true)]
 public sealed partial class EyeClosingComponent : Component
 {
+    /// <summary>
+    /// Default eyes opening sound.
+    /// </summary>
+    [ValidatePrototypeId<SoundCollectionPrototype>]
```

Comment:
As I recently found out, `[ValidatePrototypeId<SoundCollectionPrototype>]` isn't actually necessary for `private static readonly ProtoId<SoundCollectionPrototype>` fields. It does it automatically.